src/gui/qgscolorwidgets.h

```diff
@@ -60,6 +60,16 @@ class GUI_EXPORT QgsColorWidget : public QWidget
       Black //!< Black component (based on CMYK model) of color
     };
 
+    /**
+     * Specified the color component unit
+     */
+    enum class ComponentUnit
+    {
+      Raw, //!< Raw values in the range 0-255
```

Comment:
I'd lean toward "Int" here instead of "Raw", since arguably the float units are the actual raw units...

Reply:
Not sure to understand your comment, everything will be float once #58375 is merged.  "Int" looks odd because actually the components are displayed as float with decimals.

Why not?
- Plain
- Number
- NoUnit

Reply:
gentle ping

Reply:
How about something very explicit then, like "Scaled0to255"?